ruby/ruby_releases/ruby_benchmarks/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Build base image for Ruby benchmarks
+cd ../ruby_releases_base
+sudo docker build -t rubybench/ruby_releases_base:0.4 .
+cd ../ruby_benchmarks
+sudo docker build -t rubybench/ruby_releases .
```

Comment:
Actually here the problem is ruby-releases wants the tag for ruby-releases-base to be 0.4 while discourse-releases wants the tag 0.5, so I had to do this.